auto_dev/scripts/generate_command_docs.py

```diff
@@ -0,0 +1,66 @@
+"""Script to generate command documentation."""
+
+from pathlib import Path
+
+
+COMMANDS = [
+    "run",
+    "create",
+    "lint",
+    "publish",
+    "test",
+    "improve",
+    "release",
+    "metadata",
+    "fmt",
+    "scaffold",
+    "deps",
+    "convert",
+    "repo",
+    "fsm",
+    "augment",
+]
+
+TEMPLATE = """# {command_title} Command
+
+::: auto_dev.commands.{command_name}.{command_name}
+    options:
+      show_root_heading: false
+      show_source: false
+      show_signature: true
+      show_signature_annotations: true
+      docstring_style: sphinx
+      show_docstring_parameters: true
+      show_docstring_returns: false
+      show_docstring_raises: false
+      show_docstring_examples: true
+      docstring_section_style: table
+      docstring_options:
+        heading_level: 2
+        merge_init_into_class: false
+        show_docstring_attributes: false
+        show_docstring_description: true
+        show_docstring_examples: true
+        show_docstring_other_parameters: false
+        show_docstring_parameters: true
+        show_docstring_raises: false
+        show_docstring_returns: false
+        show_docstring_warns: false
+        show_docstring_yields: false
+        parameter_headings:
+          required: "Required Parameters"
+          optional: "Optional Parameters"
+"""
+
+
+def main():
```

Comment:
have this function testsed and verify there is no giff differences as part of the test.